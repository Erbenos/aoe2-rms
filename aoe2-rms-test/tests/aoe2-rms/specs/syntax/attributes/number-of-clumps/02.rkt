#lang aoe2-rms
; Create 40 forest clumps on grass terrain.

<TERRAIN-GENERATION>
(create-terrain 'FOREST
                (base-terrain 'GRASS)
                (land-percent 20)
                (number-of-clumps 40))
