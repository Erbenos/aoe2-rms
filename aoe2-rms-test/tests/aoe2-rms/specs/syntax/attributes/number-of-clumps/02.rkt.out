<TERRAIN_GENERATION>
create_terrain FOREST {
  base_terrain GRASS
  land_percent 20
  number_of_clumps 40
}
